user-interface/src/lib/components/CaseNumberInput.tsx

```diff
@@ -69,16 +85,33 @@ function CaseNumberInputComponent(props: CaseNumberInputProps, ref: React.Ref<In
     }
   }
 
+  function handleOnFocus(ev: React.FocusEvent<HTMLElement>) {
+    if (onFocus) onFocus(ev);
+  }
+
   useImperativeHandle(ref, () => ({ clearValue, resetValue, setValue, getValue, disable }));
 
+  useEffect(() => {
+    if (isDisabled && onDisable) {
+      onDisable();
+    } else if (!isDisabled && onEnable) {
+      onEnable();
+    }
+  }, [isDisabled]);
+
   return (
-    <Input
-      {...otherProps}
-      ref={forwardedRef}
-      onChange={handleOnChange}
-      onKeyDown={handleEnter}
-      includeClearButton={true}
-    ></Input>
+    <>
```

Comment:
What's the purpose of putting this input inside a fragment?

Reply:
I am not sure, it works fine without the fragment. I see your name on the commit from last week. We can just remove it